compiler/one-cmds/CMakeLists.txt

```diff
@@ -34,17 +34,17 @@ foreach(ONE_COMMAND IN ITEMS ${ONE_COMMAND_FILES})
           
 endforeach(ONE_COMMAND)
 
-set(ONE_UTILITIY_FIELS
+set(ONE_UTILITY_FIELS
```

Comment:
You've fixed to remove `I` in `UTILITY`- you have good eyes too
What I saw `ONE_UTILITY_FILES` in `FIELS` -> `FILES`